packages/react-app/pages/_app.tsx

```diff
@@ -1,45 +1,53 @@
-import { Alfajores, Celo } from "@celo/rainbowkit-celo/chains";
-import celoGroups from "@celo/rainbowkit-celo/lists";
-import { RainbowKitProvider } from "@rainbow-me/rainbowkit";
+import {
+    RainbowKitProvider,
+    connectorsForWallets,
+} from "@rainbow-me/rainbowkit";
+import { injectedWallet } from "@rainbow-me/rainbowkit/wallets";
 import "@rainbow-me/rainbowkit/styles.css";
 import type { AppProps } from "next/app";
-import { WagmiConfig, configureChains, createConfig } from "wagmi";
-import { publicProvider } from "wagmi/providers/public";
+import { http, WagmiProvider, createConfig } from "wagmi";
 import Layout from "../components/Layout";
 import "../styles/globals.css";
+import { celo, celoAlfajores } from "wagmi/chains";
 
-const projectId = process.env.NEXT_PUBLIC_WC_PROJECT_ID as string; // get one at https://cloud.walletconnect.com/app
+import { QueryClient, QueryClientProvider } from "@tanstack/react-query";
 
-const { chains, publicClient } = configureChains(
-  [Celo, Alfajores],
-  [publicProvider()]
+const connectors = connectorsForWallets(
+    [
+        {
+            groupName: "Recommended",
+            wallets: [injectedWallet],
+        },
+    ],
+    {
+        appName: "Celo Composer",
+        projectId: "044601f65212332475a09bc14ceb3c34",
```

Comment:
`projectId: "044601f65212332475a09bc14ceb3c34" || process.env.NEXT_PUBLIC_WC_PROJECT_ID as string`

If we just have the hardcoded variable people might miss that they have to exchange this for production.

Reply:
Rather than throwing an error right off the bat, this fix lets the "beginners" get to the main screen without any errors.

Production-level developers will know to change project id, we also never recommended to use celo-composer in production environment.

Reply:
@GigaHierz are we good on this?

Reply:
only concern I once had a project failing at one point because of having had that number in public for many people to use. So, I think. yes let's add it but also ask people to add their number in the .env file. if they don't they can still find this easier as the source of error.

Reply:
> my only concern is that I once had a project failing at one point because of having had that number in public for many people to use. So, I think. yes let's add it but also ask people to add their number in the .env file. if they don't they can still find this easier as the source of error.

Failing because of `projectId` is strange, is it possible to have a look at the example that failed? Can you potentially recreate that error if possible?

Reply:
https://celofoundation.slack.com/archives/C01V5UXJ6RY/p1687946055310059?thread_ts=1687551395.330579&cid=C01V5UXJ6RY

Reply:
I can't find it. looked through all my former projectIDs and I used them in many examples and tutorials and they are all working. So, tbh. it should be fine.